src/features/query-bar/components/filter/Condition.js

```diff
@@ -86,12 +91,12 @@ const Condition = ({
     };
 
     const getFormattedOptions = (options: Array<Object>): any[] => {
-        return options.map((option, optionIndex) => {
+        return options.map(option => {
             return {
                 displayText: option.displayName,
                 fieldId: option.id,
                 type: option.type,
-                value: optionIndex,
+                value: option.displayName,
```

Comment:
Is this displayName same as displayText in the other files?

Reply:
Yes